composer.json

```diff
@@ -36,5 +36,8 @@
   "scripts": {
     "lint": "vendor/bin/phpcs",
     "lint-fix": "vendor/bin/phpcbf"
+  },
+  "replace": {
+    "phpseclib/phpseclib": "*"
```

Comment:
Minor request: Could you move this below the `require` and `require-dev` directives?

Reply:
Updated in [`439e7f1`](https://github.com/google/site-kit-wp/pull/72/commits/439e7f126a5d8b8a063b8a7e2f2137ea28f92395)